cms/jinja2/templates/base.html

```diff
@@ -56,10 +57,54 @@
     }
 }
 %}
+
 {# fmt:on #}
 {% block head %}
     <link rel="stylesheet" href="{{ static('css/main.css') }}">
-{% endblock %}
+
+  <!-- Google Tag Manager -->
+    <script>
+
+        function loadGTM(){
+            (function(w,d,s,l,i){
+                w[l]=w[l]||[];
+                w[l].push({
+                    'gtm.start': new Date().getTime(),
+                    event:'gtm.js'
+                });
+                var f=d.getElementsByTagName(s)[0],
+                    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';
+                j.async=true;
+                j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;
+                f.parentNode.insertBefore(j,f);
+            })
+            (window,document,'script','dataLayer', "{{ GOOGLE_TAG_MANAGER_CONTAINER_ID }}");
```

Comment:
it is a good idea to [escape](https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#std-templatefilter-escape) these values when added like that.

```suggestion
            (window,document,'script','dataLayer', "{{ GOOGLE_TAG_MANAGER_CONTAINER_ID|escape }}");
```